addons/GME/Scripts/Game/GME/Editor/Containers/Actions/ContextActions/GME_AddGarrisonContextAction.c

```diff
@@ -0,0 +1,50 @@
+//------------------------------------------------------------------------------------------------
+[BaseContainerProps(), SCR_BaseContainerCustomTitleUIInfo("m_Info")]
+class GME_AddGarrisonContextAction : SCR_BaseContextAction
+{
+	[Attribute(defvalue: "{0B9374FB0BA34B16}PrefabsEditable/Modules/AI/E_GME_Modules_AddGarrison.et", uiwidget: UIWidgets.ResourcePickerThumbnail, desc: "Prefab name of the module to be spawned", params: "et")]
```

Comment:
Might be me not following from looking at it here on github, but is it meant to be ``"Prefab name of the module to be spawned"`` here or should it have description of the garrison module?

Reply:
I named it in a more generic manner for a potential future refactor. Basically for types of context actions that spawn a module.

Reply:
Fair